src/configurations.jl

```diff
@@ -390,6 +419,25 @@ function Base.:(+)(a::Configuration{O₁}, b::Configuration{O₂}) where {O<:Abs
     Configuration(orbitals, occupancy, states)
 end
 
+"""
+    delete!(c::Configuration, o::AbstractOrbital)
+
+Remove the orbital `o` from the configuration `c`.
```

Comment:
This is easily confused with the orbital subtraction I implemented recently. Maybe clarify the docstring to say "remove entire subshell corresponding to `o`"?

Reply:
```suggestion
Remove the entire subshell corresponding to orbital `o` from configuration `c`.
```